mev-relay-rs/src/relay.rs

```diff
@@ -176,7 +176,7 @@ fn validate_signed_block(
 ) -> Result<(), Error> {
     let header = ExecutionPayloadHeader::try_from(payload)?;
     if signed_block.message.body.execution_payload_header != header {
-        return Err(Error::UnknownBlock)
+        return Err(Error::UnknownBlock);
```

Comment:
the linter was recently moved to use `nightly` `rustfmt` to support some unstable formatting lints

you'll need to run something like `cargo +nightly fmt --all` to satisfy the linter

if you want, this command is in the `justfile`, so if you have `just` installed, then you can `just fmt`